### ISSUE TITLE
Better way for manage user sessions

### ISSUE THREAD

**Issue by tavousi** [jedireza/frame#195]
i think if we have some changes in code to when user request for login, check this user for active session with same user agent and if have, system dont create new session and reply that active session exist ... github sessions working like that !

**Comment by gregfrasco** [comment]
I have written code to do that before but didn't make it in.
https://github.com/jedireza/frame/blob/9e5a5b99f1eb79864a575459dbfff3abd4b6b60d/server/api/login.js#L87-L108

**Comment by jedireza** [comment]
Yeah we should add this. One thing I would add is not to use the `ip` as a unique signature for the device. For example my phone is the same browser/session but can be on multiple IPs per day.

**Comment by jedireza** [comment]
In the next version of Frame I've simplified session handling a bit. I've removed the `revoked` property from sessions and also decided (for now) not to include any restoring session logic (this issue) by default.

I intend for Frame to be generic. This results in less opinions forced on other people and also less to maintain. For example; we're using `hapi-auth-basic` which I don't recommend that anyone use. In 100% of my use cases I remove `hapi-auth-basic` as almost the first step.  In some cases `hapi-auth-cookie` with keep alive configured has a really good session retention experience for me.

Thanks again for getting involved.
